CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 * [ENHANCEMENT] Ingester/Querier: Optimise regexps with long lists of alternates. #8221, #8234
 * [ENHANCEMENT] Ingester: Include more detail in tracing of queries. #8242
 * [EHNAHCEMENT] Distributor: add `insight=true` to remote-write and OTLP write handlers when the HTTP response status code is 4xx. #8294
+* [BUGFIX] Distributor: prometheus retry on 5xx and 429 errors, while otlp collector only retry on 429, 502, 503 and 504, mapping other 5xx errors to the retryable ones in otlp endpoint. #8324
```

Comment:
As an end-user, I have no idea what you're saying here.

What about:

```suggestion
* [BUGFIX] OTLP handler: return only retryable HTTP status codes when we want client to retry the request. Unlike Prometheus which retries on 429 and all 5xx codes, OTLP only retries requests on 429, 502, 503 and 504. #8324
```

Reply:
which part you think is not suitable in the origin description?

Reply:
As a reader, from the original description I don't understand what has changed and why. I'm suggesting to improve message to highlight what is changed, how did it work before, and reason for the change.